mathquill4quill.js

```diff
@@ -339,7 +377,7 @@ window.mathquill4quill = function(dependencies) {
 
       if (
         tooltip.getBoundingClientRect().top -
-          quill.container.getBoundingClientRect().top <
+        quill.container.getBoundingClientRect().top <
```

Comment:
Nit: let's remove this diff to keep the history clean and focused to the minimal required changes for this feature.
```suggestion
          quill.container.getBoundingClientRect().top <
```